extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -732,7 +751,8 @@ export class AmpAnalytics extends AMP.BaseElement {
           'iframePing is only available on page view requests.');
       sendRequestUsingIframe(this.win, request);
     } else {
-      sendRequest(this.win, request, this.config_['transport'] || {});
+      this.transport_.sendRequest(this.win, request,
```

Comment:
seems `this.transport_` might haven't been initialized? might be a bug after you remove the code in your constructor?